ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/connection/actions/slack/ReadMessages.java

```diff
@@ -0,0 +1,44 @@
+package io.syndesis.qe.pages.integrations.editor.add.connection.actions.slack;
+
+import static com.codeborne.selenide.Condition.exactText;
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$;
+
+import io.syndesis.qe.fragments.common.form.Form;
+import io.syndesis.qe.pages.SyndesisRootPage;
+
+import org.openqa.selenium.By;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.SelenideElement;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class ReadMessages {
+
+    private static final class Element {
+        public static final By SELECT_DELAY = By.cssSelector("[data-testid='delay-duration']");
+    }
+
+    public void selectChannel(String channel) {
```

Comment:
Could this be made more generic? I mean not just as the "chanel-list" util method, but for all datalist selections?